File: Aula 03/Python/desafio.py
print(" --- Simulador Financeiro (Python) --- ")

# Entradas
ganho = int(input("Informe o Ganho: "))
investimento = int(input("Informe o investimento: "))

#bloqueio para evitar erro no calculo
if investimento == 0:
    print("Investimento não pode ser igual a 0")
    exit()

#processamento
roi = ((ganho - investimento) / investimento) * 100
print(f'Resultado: {roi}')

File: Aula 03/Java/desafio.java
import java.util.Locale;
import java.util.Scanner;

public class desafio {
    public static void main(String[] args) {
        // Define o ponto como separador decimal
        Locale.setDefault(Locale.US);

        //Abre o teclado
        Scanner scanner = new Scanner(System.in);

        System.out.println(" --- Simulador Financeiro (Java) --- ");

        // Entradas
        System.out.println("Informe o Ganho: ");
        double ganho = scanner.nextDouble();

        System.out.println("Informe o Investimento: ");
        double investimento = scanner.nextDouble();

        if(investimento == 0) {
            System.out.println("Investimento não pode ser igual a zero");
            scanner.close();
            return;
        }
        // Processamento do cálculo e definição da variável de resultado
        double roi = ((ganho - investimento) / investimento) * 100;
        
        // Saída
        System.out.printf("Resultado: %.2f%n", roi);

        scanner.close();
    }
}
